bin/ch-run.c

```diff
@@ -314,6 +321,9 @@ static error_t parse_opt(int key, char * arg, struct argp_state * state)
    case -2:
       as->private_home = true;
    break;
```

Comment:
While you're in there, would you indent this `break` to match?

Reply:
Done as part of the stashed commit.